### PR TITLE
Clarify use of (rel)-pool values

### PR DESCRIPTION
**Pull request by calnan**
(rel)-pool values should not be used for affects versions, but can, along with tbd, be used in the fix version

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Iris Clark](https://openjdk.org/census#iris) (@irisclark - **Reviewer**)
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - Author)
 * [Jesper Wilhelmsson](https://openjdk.org/census#jwilhelm) (@JesperIRL - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/guide.git pull/134/head:pull/134` \
`$ git checkout pull/134`

Update a local copy of the PR: \
`$ git checkout pull/134` \
`$ git pull https://git.openjdk.org/guide.git pull/134/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 134`

View PR using the GUI difftool: \
`$ git pr show -t 134`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/guide/pull/134.diff">https://git.openjdk.org/guide/pull/134.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/guide/pull/134#issuecomment-2284616870)

### CODE REVIEW

**Review by irisclark — APPROVED** [review]

File: src/guide/jbs-jdk-bug-system.md [hunk 1]
```diff
@@ -219,9 +219,9 @@ Now that the issue is in the right component and has the basic information, the
     * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
       * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
     * For enhancements the [Affects Version/s]{.jbs-field} should be empty unless you feel that the enhancement is only relevant to a particular release family, and shouldn't go into a future mainline release.
-    * [Affects Version/s]{.jbs-field} should never use any of the "special" values available in JBS like [tbd]{.jbs-value}, [na]{.jbs-value}, [unknown]{.jbs-value} or similar. Only actual JDK release numbers or [-pool]{.jbs-value} versions should be used.
+    * [Affects Version/s]{.jbs-field} should never use any of the "special" values available in JBS like [tbd]{.jbs-value}, [na]{.jbs-value}, [unknown]{.jbs-value}, [(Rel)-pool]{.jbs-value} or similar. Only actual JDK release numbers versions should be used, if you want to reflect that an issue is relevant to an older release use the family release value [8]{.jbs-value}, [17]{.jbs-value}.
```
Comment (dholmes-ora) [inline src/guide/jbs-jdk-bug-system.md:222]: > Only actual JDK release numbers versions 

should be

> Only actual JDK release numbers

The "versions" came from "or pool versions".

Comment (calnan) [reply src/guide/jbs-jdk-bug-system.md]: fixed

File: src/guide/jbs-jdk-bug-system.md [hunk 2]
```diff
@@ -219,9 +219,9 @@ Now that the issue is in the right component and has the basic information, the
     * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
       * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
     * For enhancements the [Affects Version/s]{.jbs-field} should be empty unless you feel that the enhancement is only relevant to a particular release family, and shouldn't go into a future mainline release.
-    * [Affects Version/s]{.jbs-field} should never use any of the "special" values available in JBS like [tbd]{.jbs-value}, [na]{.jbs-value}, [unknown]{.jbs-value} or similar. Only actual JDK release numbers or [-pool]{.jbs-value} versions should be used.
+    * [Affects Version/s]{.jbs-field} should never use any of the "special" values available in JBS like [tbd]{.jbs-value}, [na]{.jbs-value}, [unknown]{.jbs-value}, [(Rel)-pool]{.jbs-value} or similar. Only actual JDK release numbers should be used, if you want to reflect that an issue is relevant to an older release use a family release value: [8]{.jbs-value}, [17]{.jbs-value}.
```
Comment (dholmes-ora) [inline src/guide/jbs-jdk-bug-system.md:222]: Shouldn't you use the actual release version where the existence of the issue was detected? The above sounds to me like you are suggesting to use e.g. 8 even if the issue is actually only applicable for 8uX.

Comment (calnan) [reply src/guide/jbs-jdk-bug-system.md]: Correct. the actual update version should be used if known.  Spoke with Jesper and he made some suggested changes to the text should should clarify that

**Review by dholmes-ora — APPROVED** [review]
Looks fine. Thanks

**Review by irisclark — APPROVED** [review]

**Review by JesperIRL — APPROVED** [review]

**Pull request closed without merging**